pkts-sip/src/main/java/io/pkts/packet/sip/impl/SipMessageBuilder.java

```diff
@@ -458,8 +473,8 @@ private short addHeader(final SipHeader header) {
             this.viaHeaders.clear();
             for (final ViaHeader via : vias) {
                 this.viaHeaders.add(via);
+                viaHeader = via;
```

Comment:
bug; need to remember the first via header...

Reply:
btw, if you don't mind, could you add some unit tests to ensure that we do store the first via header. And for Route and RecordRoute headers too

Reply:
doesn't testViaHeaders do that, for via of course?

Reply:
need to check. perhaps it doesn't check the one that is "remembered" as the first position... even so, if you compare with the old code, seems like a different behavior.

Reply:
did you check latest commit?